libraries/lib-wave-track/WaveClip.cpp

```diff
@@ -659,23 +659,17 @@ void WaveClip::WriteXML(XMLWriter &xmlFile) const
 }
 
 /*! @excsafety{Strong} */
-bool WaveClip::Paste(
-   double t0, const WaveClip& other,
-   ProgressReporter reportProgress)
+bool WaveClip::Paste(double t0, const WaveClip& other)
 {
    if (GetWidth() != other.GetWidth())
       return false;
 
+   assert(GetStretchRatio() == other.GetStretchRatio());
```

Comment:
If the function can already return false for failure, and uses of it are checked, maybe there should not be an assertion.

Reply:
Yeah sure.